### PR TITLE
Ztri 32 markdown to jsx

### PR DESCRIPTION
**Pull request by Joepock123**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
 		"@typescript-eslint/parser": "^3.2.0",
 		"airtable": "^0.8.1",
 		"inquirer": "^7.1.0",
+		"markdown-to-jsx": "^6.11.4",
```
Comment (VatsKan) [inline package.json:26]: i see youve added another dependency interesting

Comment (Joepock123) [reply package.json:26]: How about dat

File: src/components/Pages/WorkshopContent/WorkshopContent.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+// import Markdown from 'markdown-to-jsx';
+
+import * as SC from './style';
```
Comment (VatsKan) [inline src/components/Pages/WorkshopContent/WorkshopContent.js:5]: very clear what SC means

Comment (Joepock123) [reply src/components/Pages/WorkshopContent/WorkshopContent.js:5]: [Link to the issue which explains the SC syntax and use of a style.js for each component](https://github.com/fac19/Zaatari-Radio/issues/2)

File: src/styles/variables.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ export const offWhite = '#fbf9fa';
 export const medButtonHeight = '42px';
 export const smallButtonHeight = '38px';
 export const titleBarVerticalPadding = '8px';
+export const mainContainerWidth = '700px';
```
Comment (VatsKan) [inline src/styles/variables.js:13]: nice use of relative units

**Comment by VatsKan** [comment]
well done folks

**Pull request merged**